# ==============================================================================
# FIXING COMMIT DIFF
# ==============================================================================
# File: gfx/config/gfxVars.h
# Commit: fe7075f48cf0
# Full Hash: fe7075f48cf031d5686b3268a0e738581721b386
# Author: Andrew Osmond <aosmond@mozilla.com>
# Date: 2020-07-23 03:20:25
# Description:
#   Bug 1650748 - Prevent DMABUF usage with GLX. r=nical
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D84256
# ==============================================================================

diff -r 36b357364a21 -r fe7075f48cf0 gfx/config/gfxVars.h
--- a/gfx/config/gfxVars.h	Wed Jul 22 21:00:42 2020 +0000
+++ b/gfx/config/gfxVars.h	Wed Jul 22 18:58:11 2020 +0000
@@ -65,7 +65,8 @@
   _(ForceSubpixelAAWherePossible, bool, false)                     \
   _(DwmCompositionEnabled, bool, true)                             \
   _(FxREmbedded, bool, false)                                      \
-  _(UseAHardwareBufferContent, bool, false)
+  _(UseAHardwareBufferContent, bool, false)                        \
+  _(UseEGL, bool, false)
 
 /* Add new entries above this line. */
 